scripts/build/clean.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Cleans top-level build directories (for OSS and internal processes),
+ * dist subfolders under packages, and .js[.map] files in package directories.
+ */
+
+const {exec} = require('child_process');
+const fs = require('fs');
+
+const {sync: globSync} = require('glob');
+
+if (require.main === module) {
+  main();
+}
+
+function main() {
+  removeDirectory('build');
+  removeDirectory('.rewrite-tmp');
+  removeFilesOfType('css');
+  removeFilesOfType('js');
+  removeFilesOfType('d.ts');
+  removeFilesOfType('map');
+}
+
+function removeDirectory(directory) {
+  exec(`del-cli ${directory}/ ${directory}`, (err) => {
```

Comment:
My bad - it looks like we want https://github.com/sindresorhus/del instead. That'll give us a programmatic API so we don't have to invoke a subshell.